assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -27,25 +27,33 @@ import isPlainObject from 'lodash/isPlainObject';
  */
 import API from 'googlesitekit-api';
 import Data from 'googlesitekit-data';
+import { deleteItem, getItem, setItem } from '../../../googlesitekit/api/cache';
 import { STORE_NAME } from './constants';
 import { createFetchStore } from '../../data/create-fetch-store';
 import { actions as errorStoreActions } from '../../data/create-error-store';
-const { commonActions, createRegistrySelector } = Data;
+const { commonActions, createRegistryControl, createRegistrySelector } = Data;
 const { receiveError, clearError } = errorStoreActions;
 
+const CACHE_KEY_NAME = 'fetchGetUserInputSettingsStore.fetchGetUserInputSettingsStore';
```

Comment:
Why this overly cryptic/specific cache key name with a duplicated segment? It makes me think that name is somehow required due to how it integrates with `fetchGetUserInputSettingsStore`. We can just give it an arbitrary name like `userInputSettings`.

Reply:
Fair point—there's no reason for it other than specificity, but `userInputSettings` should be fine 👍🏻